docs/source/examples/cirq-ibmq-backends.myst

```diff
@@ -0,0 +1,264 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(label-cirq-zne-example)=
+# Error mitigation on IBMQ backends with Cirq
+
+
+This tutorial shows an example of how to mitigate noise on IBMQ
+backends with the Cirq frontend, broken down in the following steps.
+
+- [Error mitigation on IBMQ backends with Cirq](#error-mitigation-cirq-on-ibmq-backends)
+  - [Settings](#settings)
+  - [Setup: Defining a circuit](#setup-defining-a-circuit)
+  - [High-level usage](#high-level-usage)
+  - [Options](#options)
+  - [Lower-level usage](#lower-level-usage)
+
++++
+
+## Settings
+
+```{code-cell} ipython3
+from mitiq import zne
+from mitiq.interface.mitiq_qiskit.qiskit_utils import initialized_depolarizing_noise
+```
+
+**Note:** If `USE_REAL_HARDWARE` is set to `False`, a classically simulated noisy backend is used instead of a real quantum computer.
+
+```{code-cell} ipython3
+USE_REAL_HARDWARE = False
+```
+
+(examples/cirq-ibmq-backends/setup-defining-a-circuit)=
+## Setup: Defining a circuit
+
++++
+
+For simplicity, we'll use a random single-qubit circuit with ten gates that compiles to the identity, defined below.
+
+(examples/cirq-ibmq-backends/cirq-frontend)=
+### Cirq frontend
+
+It isn't necessary to use Qiskit frontends (circuits) to run on IBM backends. We can use conversions in
+Mitiq to use any supported frontend with any supported backend. Below, we show how to run a Cirq circuit on an
+IBMQ backend.
+
+First, we define the Cirq circuit.
+
+```{code-cell} ipython3
+import cirq
+
+qbit = cirq.LineQubit(0)
+cirq_circuit = cirq.Circuit([cirq.X(qbit)] * 10, cirq.measure(qbit))
+print(cirq_circuit)
+```
+
+Now, we simply add a line to our executor function which converts from a Cirq circuit to a Qiskit circuit.
```

Comment:
cirq user may not be familiar with our "executor" terminology. Can we add a link to our docs explaining the idea: https://mitiq.readthedocs.io/en/stable/guide/executors.html?

Reply:
Added a reference in the section where `cirq_ibm_executor` is defined.